src/mopi/blocks/io.py

```diff
@@ -60,11 +60,29 @@ def pickle_saving(
 
 
 def export_pipeline(name: str, pipeline: "Pipeline") -> None:
+
+    blocks = [
+        block
+        for source_type in pipeline.get_datasource_types()
+        for block in flatten(pipeline.children(source_type))
+    ]
+
+    for block in blocks:
+        if hasattr(block, "model") and block.model is not None:
+            block.model = None
+        if (
+            hasattr(block, "dataloader")
+            and hasattr(block.dataloader, "data")
+            and block.dataloader.data is not None
+        ):
+            block.dataloader.data = None
+
```

Comment:
The current solution involves us saving an unloaded pipeline. I would keep this part in case we assign a model in a blocks ``__init__`` rather than in ``load``. 
However, that would be against our current pattern, so I could also just delete this part.